content/en-us/reference/engine/classes/CaptureService.yaml

```
@@ -15,8 +15,8 @@ deprecation_message: ''
 properties: []
 methods:
   - name: CaptureService:CaptureScreenshot
-    summary: ''
-    description: ''
+    summary: 'Take a screenshot associated with the returned contentId.'
+    description: 'Take a screenshot that can be identified by the returned contentId. This function does not save the screenshot to the gallery.'
```

```suggestion
    description: 'Take a screenshot that can be identified by a temporary contentId. This function does not save the screenshot to the gallery.'
```

Might be more clear if it specifies that the content is temporary.